root/inc/module-install.html

```diff
@@ -0,0 +1,35 @@
+
+<script type="text/javascript">
+	function show_install_instructions() {
+		jQuery('#install_module').modal('show');
+	}
+</script>
+
+<div id="install_module" class="modal fade">
+	<div class="modal-dialog">
+      <div class="modal-content">
+        <div class="modal-header">
+          <button type="button" class="close" data-dismiss="modal" aria-hidden="true">&times;</button>
+          <h4 class="modal-title">Module Install Instructions</h4>
+        </div>
+        <div class="modal-body">
+		<p>To install <%  module.documentation or module.module.0.name %>, simply copy and paste either of the commands in to your terminal</p>
+			<p>CPANminus</p>
+			<pre class="bash: js">
+				sudo cpanm install <%  module.documentation or module.module.0.name %>
```

Comment:
I don't think we want to default to installing modules via `sudo`, since that would mess with the system Perl.  We could either drop the `sudo` completely or offer it as an alternative with the caveat that you really should only do this if you know what you're doing.